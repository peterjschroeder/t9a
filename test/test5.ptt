
C:\Users\Manuel\Documents\GitHub\9th-age-latex\test>"C:\xpdf\bin64\pdftotext.exe" C:\Users\Manuel\Documents\GitHub\9th-age-latex\test\test5.pdf C:\Users\Manuel\Documents\GitHub\9th-age-latex\test\test5_parsing.par 
