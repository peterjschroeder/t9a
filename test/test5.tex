\documentclass[10pt]{article}
\usepackage{lipsum}
\usepackage{hyperref}

\begin{document}

\section{test}

\lipsum

\begin{itemize}
\item test
\item test2
\item \hypertarget{test}{ici}
\item test 3
\end{itemize}

\phantomsection\label{testC} Et là, \hypertarget{testB}{ça marche ?}

\section{test2}

\label{lalala}

egzqgerhqrgqergsreerh

\hypertarget{arg}{ARRRRRRRRRRRRGGGHHHHHHHHHH}

\newpage

\hyperlink{test}{ça part par là}

\hyperlink{testB}{ça part par là}

\hyperref[testC]{et là ?}

\hyperref[lalala]{gggrgsgsrgssgr}

\hyperlink{arg}{AAAAAAAARGGGHHHHHHHHHHHHHH}





\end{document}