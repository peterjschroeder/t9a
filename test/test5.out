\BOOKMARK [1][-]{section.1}{test}{}% 1
\BOOKMARK [1][-]{section.2}{test2}{}% 2
